utils/order.utils.js

```diff
@@ -0,0 +1,35 @@
+const { ORDER_ITEM_NOT_VALID } = require('../error-messages/orders.messages');
+const crypto = require('crypto');
+
+// this method should return totalItemsPrice + nova poshta delivery price
+function novaPoshtaDeliveryPrice(data) {
+  // need backend for sizes
+}
+
+// this method should return totalItemsPrice + ukr poshta delivery price
+function ukrPoshtaDeliveryPrice(data) {
+  // need backend for sizes
+}
+
+function calculateTotalPriceToPay(data) {
+  switch (data.delivery.sentBy) {
+    case 'Nova Poshta':
+      return novaPoshtaDeliveryPrice();
+
+    case 'Ukr Poshta':
+      return ukrPoshtaDeliveryPrice();
+
+    default: {
+      return data.totalItemsPrice;
```

Comment:
pls, change this values to NOVAPOST, UKRPOST and SELFPICKUP in cases from qraphQL enum. And also try to implement this methods to avoid errors